code/style.css

```
@@ -1,5 +1,333 @@
+  font-family: "Domine", serif;
+}
+h3 {
+  text-align: center;
+  color: rgb(161, 13, 40, 0.9);
+  padding: 15px;
+  margin: 0px;
+  font-family: "Domine", serif;
+}
+h4 {
+  text-align: center;
+  padding: 10px;
+  margin: 0px;
+  font-family: "Domine", serif;
+}
+p {
+  text-align: center;
+  padding-left: 20px;
+  padding-right: 20px;
+  font-family: "Open Sans", sans-serif;
+}
+.container {
+  max-width: 1200px;
+  margin-left: auto;
+  margin-right: auto;
+}
+.row {
+  display: flex;
+  margin-bottom: 50px;
+  gap: 50px;
+}
+
+.center {
+  justify-content: center;
+  gap: 40px;
+}
+
+.column {
+  flex: 1 1 0%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.column-50 {
+  flex: 1 1 0%;
+  display: flex;
+  align-items: center;
+}
+.Trauzeuge-left {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.round-image {
+  height: 200px;
+  width: 200px;
+  border-radius: 50%;
+  object-fit: cover;
+  object-position: top;
+  text-align: center;
+  padding: 10px;
+}
+a {
+  color: rgb(161, 13, 40);
+  text-align: center;
+}
+p {
+  color: black;
+}
+
+.wunsch-left {
+  width: 33%;
+}
+
+.image-wunsch {
+  width: 175px;
+  padding: 10px;
+  border-radius: 10px 25% 20px;
+}
+
+.material-icons {
+  font-family: "Material Icons";
+  font-weight: normal;
+  font-style: normal;
+  font-size: 24px; /* Preferred icon size */
+  display: inline-block;
+  line-height: 1;
+  text-transform: none;
+  letter-spacing: normal;
+  word-wrap: normal;
+  white-space: nowrap;
+  direction: ltr;
+}
+.material-icons-round.white {
+  color: #ffffff;
+}
+.icons {
+  background-color: rgb(161, 13, 40);
+  padding: 20px;
+  border: rgb(161, 13, 40);
+  height: 50px;
+  width: 50px;
+  border-radius: 50%;
+  margin: 20px;
+  display: flex;
+  flex-shrink: 0;
+  align-items: center;
+  justify-content: center;
+}
+.icons .icon {
+  font-size: 36px;
+}
+
+ul {
+  list-style: none;
+}
+
+.menu {
+  display: inline-flex;
+  margin-right: 10px;
+  padding-bottom: 10px;
+}
+
+li {
+  display: flex;
+  justify-content: center;
+  gap: 0px;
+}
+
+.schedule {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  padding-left: 0;
+}
+.schedule-point {
+  display: flex;
+  padding-bottom: 20px;
+}
+.time {
+  font-size: large;
+}
+.schedule-text {
+  margin-top: 0px;
+}
+
+.ablauf-text {
+  flex: 0 0 295px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+.empty-map {
+  height: 200px;
+  width: 400px;
+  flex-shrink: 0;
+}
+iframe {
+  height: 200px;
+  width: 400px;
+}
+
+/*.Wunschliste {
+  flex: 1 1 auto;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  gap: 100px;
+}*/
+
+/*.wunsch-container {
+  display: flex;
+  flex-direction: column;
+  flex: 0 0 230px;
+  align-items: center;
+}*/
+.Wunschliste {
+  display: flex;
+  flex-wrap: wrap;
+  padding-left: 0px;
+}
+
+.wunsch-text {
+  max-width: 250px;
+}
+.wishlist {
+  padding-bottom: 20px;
+  width: 33%;
+}
```

To simplify the `li` structure as mentioned, you'd have to make the following changes:
```suggestion
.wishlist {
  flex-direction: column;
  align-items: center;
  padding-bottom: 20px;
  width: 33%;
}
```